pkg/dump/pipeline/metadata.go

```diff
@@ -0,0 +1,28 @@
+package pipeline
+
+import (
+	"context"
+
+	"github.com/DataDog/KubeHound/pkg/collector"
+	"github.com/DataDog/KubeHound/pkg/dump/writer"
+)
+
+type MetadataIngestor struct {
+	buffer map[string]collector.Metadata
+	writer writer.DumperWriter
+}
+
+func NewMetadataIngestor(ctx context.Context, dumpWriter writer.DumperWriter) *MetadataIngestor {
+	return &MetadataIngestor{
+		buffer: make(map[string]collector.Metadata),
+		writer: dumpWriter,
+	}
+}
+
+func (d *MetadataIngestor) DumpMetadata(ctx context.Context, metadata collector.Metadata) error {
+	data := make(map[string]*collector.Metadata)
```

Comment:
Is it expected that this is a pointer to the collector metadata ? :sweat_smile:

The buffer above is `make(map[string]collector.Metadata),`

Reply:
no it is not 😅